### PR TITLE
GT config fix for fluid pumping

### PR DESCRIPTION
**Pull request by 0lafe**
I rewrote some of the GT config for fluids to remove Nq pumping, as well as add a lot of other options. Lava/H2S in the nether, infinite oil veins on T3 planets, N2 gen on mars, sulfuric light/heavy fuel pumping on T5s, O2 gen on T3, argon/He gen on T4, toluene/liquid air/helium on T6, pure Cl on T7, and F/H2/Rn on T8. Idk how PR's really work, but I think I did it right?

### CODE REVIEW

**Comment by Dream-Master** [comment]
i see H2 and SN.  Are the stuff is tested @0lafe  ?

**Comment by ScriptedPiky** [comment]
F on T8 is a little odd as well. How come F and H2 are T8, while toluene and helium are on T6? F and H2 being on T8 feels as if it's gated too far. More different types of fluids would also be nice, since technically, some planets can have molten metal (Venus having lead). However, that can be adjusted to be less common and less potent

**Comment by 0lafe** [comment]
H2 and F are more useful fluids than He or toluene, having a semi unlimited fast creation rate on them is more useful. He has He fusion which is also great for power, and toluene is mostly able to be produced much beyond your need before getting to the pumping. H2 and F however have slow/non infinite methods of production, and are impacted more with a pump option.

**Comment by richardhendricks** [comment]
https://github.com/GTNewHorizons/NewHorizons/issues/4388

**Comment by richardhendricks** [comment]
https://github.com/GTNewHorizons/NewHorizons/issues/4360

**Comment by Dream-Master** [comment]
@0lafe  @richardhendricks @bartimaeusnek any news here ?  When the config is done i will merge it.

File: config/GregTech/GregTech.cfg [hunk 1]
```diff
@@ -224,6 +223,45 @@ undergroundfluid {
...
+		
+		lava {
+            # Chance generating (weighted chance!, there will be a fluid in chunk always!)
+            I:Chance=100
+
+            # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
+            I:DecreasePerOperationAmount=1
+
+            # Max amount generation (per operation, sets the VeinData) 80000 MAX
+            I:MaxAmount= 50
```
Comment (richardhendricks) [inline config/GregTech/GregTech.cfg]: Why such a small amount?  If someone needs large quantities of lava and wants to get it from the Nether, this seems really low.

File: config/GregTech/GregTech.cfg [hunk 2]
```diff
@@ -649,97 +725,101 @@ undergroundfluid {
...
-            I:Chance=20
+            I:Chance=100
 
             # Decrease per operation (actual fluid gained works like (Litre)VeinData/5000)
-            I:DecreasePerOperationAmount=5
+            I:DecreasePerOperationAmount=1
 
             # Max amount generation (per operation, sets the VeinData) 80000 MAX
-            I:MaxAmount=20
+            I:MaxAmount=800
```
Comment (richardhendricks) [inline config/GregTech/GregTech.cfg]: Fluorine at 800 is crazy high.

**Comment by richardhendricks** [comment]
Merge denied, will be redoing fluid changes after fluid drill is updated.

**Comment by Dream-Master** [comment]
i reopen it and let it open untill it is nerfed/buffed right @0lafe @richardhendricks

**Pull request closed without merging**